app/main.py

```diff
@@ -1 +1,48 @@
-#  write your code here
+from __future__ import annotations
+
+
+class Animal:
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry is True:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = not self.is_hungry
```

Comment:
this is technically correct, but in this case you definitely need to set only False, so it'll be more explicit for other devs.
"not self.is_hungry" is often used when it's possible to switch between True and False anytime. For example when you like a photo in instagram. There is a field "is_liked". Each time user clicks on a heart, it should set vice-versa value. Like -> unlike, unlike -> Like. In your case only the first case is possible ;)